packages/block-editor/src/components/media-upload/index.native.js

```diff
@@ -93,7 +94,11 @@ export class MediaUpload extends React.Component {
 	}
 
 	getMediaOptionsItems() {
-		const { allowedTypes = [], multiple = false } = this.props;
+		const {
+			allowedTypes = [],
+			multiple = false,
+			onlyMediaLibrary,
```

Comment:
If this is a temporary thing until we have the full implementation, it would be nice to flag it as `__experimental` or `__unstable`

Reply:
I like that, I'll use one of those suggestions and if for any future feature we need this functionality we could just make it as a final prop instead of experimental

Reply:
Updated the naming https://github.com/WordPress/gutenberg/pull/19722/commits/a61cbb64fa7f1952b245b088818fbc5b0bfa3ac6#diff-61fa0159776aa47e1331c57eb5ccced8